deepmd/entrypoints/transfer.py

```diff
@@ -89,6 +100,36 @@ def load_graph(graph_name: str) -> tf.Graph:
         tf.import_graph_def(graph_def, name="")
         return graph
 
+def modify_const_op(new_graph_def: tf.Graph) -> tf.Graph:
+    """modify natoms to constant.
+
+    Parameters
+    ----------
+    new_graph : tf.Graph 
+        orginal new graph
+    Returns :
+        natoms transfer to a const op for Ascend platform
+    -------
```

Comment:
This style is wrong. See numpydoc style guide for reference. https://numpydoc.readthedocs.io/en/latest/format.html#returns

Reply:
Thanks, we have update the styles.

Reply:
The line where I commented was not updated.

Reply:
Thanks, we updated the refernce.